src/index.html

```diff
@@ -2,11 +2,325 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span>
+        <a href="#home">
+          <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+        </a>
+      </span>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a class="nav__link is-active" href="#apple">APPLE</a></li>
+          <li class="nav__item"><a class="nav__link" href="#samsung">SAMSUNG</a></li>
+          <li class="nav__item"><a class="nav__link" href="#smartphones">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#laptops"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item"><a class="nav__link" href="#gadgets">ГАДЖЕТЫ</a></li>
+          <li class="nav__item"><a class="nav__link" href="#tablets">ПЛАНШЕТЫ</a></li>
+          <li class="nav__item"><a class="nav__link" href="#photo">ФОТО</a></li>
+          <li class="nav__item"><a class="nav__link" href="#video">ВИДЕО</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="product">
```

Comment:
I think that you can add this class to <main> so you wouldn't have one extra div